### PR TITLE
fix: [CDS-73426]: fixed the dropdown selection value on cloudformatio…

### PR DESCRIPTION
**Pull request by sahithikolichala**
…n stack step

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cd/components/PipelineSteps/CloudFormation/CreateStack/CreateStack.tsx [hunk 1]
```diff
@@ -245,8 +249,11 @@ export class CFCreateStack extends PipelineStep<CreateStackStepInfo> {
     const config = data?.spec?.configuration
     let capabilities, skipOnStackStatuses, parameters
     capabilities = config?.capabilities
+
+    //console.log(capabilities, 'init')
```
Comment (ishantmehtaharness) [inline src/modules/75-cd/components/PipelineSteps/CloudFormation/CreateStack/CreateStack.tsx:253]: we can remove console.log

**Review by ishantmehtaharness — APPROVED** [review]
rest looks good

**Comment by sahithikolichala** [comment]
This is draft pr

**Comment by Sagar-Rajput7** [comment]
I tested the changes (locally from your branch) - and seems the issue is not resolved yet.
- After saving - as soon as we check the fields again (label: value) gets added in the yaml (which is not supported by BE)
- These multi-select fields get reset as soon as we change values in any other fields ( in the below video I enter random values in the tag script.

(please see the below video for reference)

https://github.com/harness/harness-core-ui/assets/98325173/e278b7ce-0037-4416-9f40-feffdc0dfdfd

**Pull request closed without merging**